citesphere/pom.xml

```diff
@@ -24,8 +24,10 @@
         <javers.version>6.2.3</javers.version>
         <spring.kafka.version>2.2.6.RELEASE</spring.kafka.version>
         <spring-social-zotero.version>0.12</spring-social-zotero.version>
-        <citesphere.messages.version>0.4</citesphere.messages.version>
+        <citesphere.messages.version>0.5</citesphere.messages.version>
         <citesphere.model.version>1.19</citesphere.model.version>
+        <crossref-connect-version>0.3-SNAPSHOT</crossref-connect-version>
```

Comment:
there is not 0.3-SNAPSHOT version. if you made changes to this plugin, there needs to be a pr (and that needs to be linked here).

Reply:
Changed it to 0.2 as citesphere-import